crates/context/src/lib.rs

```diff
@@ -238,6 +238,30 @@ impl ContextManager {
         Ok(ids)
     }
 
+    // Iterate over all identities in a context (from members and mine)
+    // and return only public key of identities which contains private key (in value)
+    // If there is private key then it means that identity is mine.
+    pub fn get_context_identities(
+        &self,
+        context_id: calimero_primitives::context::ContextId,
+    ) -> eyre::Result<Vec<PublicKey>> {
+        let handle = self.store.handle();
+
+        let mut iter = handle.iter::<calimero_store::key::ContextIdentity>()?;
+        let mut ids = Vec::<PublicKey>::new();
+
+        for (k, v) in iter.entries() {
+            let (k, v) = (k?, v?);
+
+            if k.context_id() == context_id {
+                if v.private_key.is_some() {
+                    ids.push(PublicKey::from(k.public_key()));
```

Comment:
can we use this here:

```suggestion
                    ids.push(PublicKey(k.public_key()));
```

just so it breaks when I change the signature of `PublicKey` to be consistent with the pattern established with `ContextId`, `ApplicationId` and `BlobId`

the reason for this is `k.public_key()` "should" return `PublicKey` already, but currently doesn't

that'll change soon